transaction.go

```diff
@@ -97,6 +97,30 @@ type TransactionRequest struct {
 	CustomFields       customfields.CustomFields `xml:"custom-fields,omitempty"`
 }
 
+type TransactionCloneRequest struct {
+	XMLName            string                    `xml:"transaction-clone"`
+	CustomerID         string                    `xml:"customer-id,omitempty"`
+	Type               string                    `xml:"type,omitempty"`
+	Amount             *Decimal                  `xml:"amount"`
+	OrderId            string                    `xml:"order-id,omitempty"`
+	PaymentMethodToken string                    `xml:"payment-method-token,omitempty"`
+	PaymentMethodNonce string                    `xml:"payment-method-nonce,omitempty"`
+	MerchantAccountId  string                    `xml:"merchant-account-id,omitempty"`
+	PlanId             string                    `xml:"plan-id,omitempty"`
+	CreditCard         *CreditCard               `xml:"credit-card,omitempty"`
+	Customer           *Customer                 `xml:"customer,omitempty"`
+	BillingAddress     *Address                  `xml:"billing,omitempty"`
+	ShippingAddress    *Address                  `xml:"shipping,omitempty"`
+	TaxAmount          *Decimal                  `xml:"tax-amount,omitempty"`
+	TaxExempt          bool                      `xml:"tax-exempt,omitempty"`
+	DeviceData         string                    `xml:"device-data,omitempty"`
+	Options            *TransactionOptions       `xml:"options,omitempty"`
```

Comment:
Comparing the `TransactionsOptions` struct to the [TransactionOptionsCloneRequest](https://github.com/braintree/braintree_java/blob/acc7c9ada19e09f6181dd501657fc7f4f1b7afcb/src/main/java/com/braintreegateway/TransactionOptionsCloneRequest.java) in the Braintree Java SDK, I see that many of the fields in the `TransactionsOptions` struct are not intended to be in a clone options request. 

Can you create a new struct `TransactionCloneOptions` similar to `TransactionOptions` but with a single field `SubmitForSettlement bool`? That's the only field listed in the Braintree Java SDK.